cdap-gateway/src/main/java/co/cask/cdap/gateway/router/NettyRouter.java

```diff
@@ -236,6 +220,10 @@ public ChannelPipeline getPipeline() throws Exception {
             pipeline.addLast("ssl", sslHandlerFactory.create());
           }
           pipeline.addLast("tracker", connectionTracker);
+          // idle state handler
+          pipeline.addLast("idle-state-handler",
```

Comment:
Calling this `idle-event-generator` might make the intent for `IdleStateHandler` clearer.